community/php7/APKBUILD

```diff
@@ -1,72 +1,71 @@
+depends_dev="$pkgname"
+replaces="$_pkgreal $_pkgreal-cli"
+provides="$replaces $pkgname-cli"
 makedepends="autoconf bison re2c apache2-dev libxml2-dev libxslt-dev libzip-dev bzip2-dev zlib-dev
 	aspell-dev enchant-dev expat-dev pcre-dev curl-dev gmp-dev icu-dev imap-dev
 	libical-dev libressl-dev openldap-dev net-snmp-dev db-dev krb5-dev gdbm-dev sqlite-dev
 	freetds-dev mariadb-dev postgresql-dev unixodbc-dev freetype-dev tidyhtml-dev libxpm-dev
-	libpng-dev libwebp-dev libjpeg-turbo-dev libmcrypt-dev gsoap-dev recode-dev
-	libedit-dev paxmark gettext-dev
+	libpng-dev libwebp-dev libjpeg-turbo-dev libmcrypt-dev recode-dev libedit-dev gettext-dev
 	"
 source="http://php.net/distributions/$_pkgreal-$pkgver.tar.bz2
-	$pkgname-fpm.initd
-	$pkgname-fpm.logrotate
 	$pkgname-module.conf
-	install-pear.patch
+	$pkgname-fpm.logrotate
+	$pkgname-fpm.initd
+	$pkgname-fpm.patch
+	$pkgname-includedir.patch
 	tidy-buffio.patch
-	includedir.patch
+	install-pear.patch
 	fix-asm-constraints-in-aarch64-multiply-macro.patch
-	pid_log.patch
 	getrandom.patch
 	"
-builddir="$srcdir/$_pkgreal-$pkgver"
-
-_exts="bcmath bz2 calendar ctype curl dba dom enchant exif ftp gd gettext gmp iconv imap intl json
+# unimplemented extensions: com_dotnet interbase oci8 pdo_firebird pdo_oci
+_extensions="bcmath bz2 calendar ctype curl dba dom enchant exif ftp gd gettext gmp iconv imap:1 intl json
 	ldap mbstring mcrypt mysqli mysqlnd odbc opcache openssl pcntl pdo pdo_dblib pdo_mysql
-	pdo_odbc pdo_pgsql pdo_sqlite pgsql phar:phar posix pspell session shmop snmp soap
-	sockets sqlite3 sysvmsg sysvsem sysvshm tidy wddx xml xmlreader xmlrpc xsl zip zlib
-	"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-apache2 $pkgname-phpdbg $pkgname-embed
-	$pkgname-litespeed $pkgname-cgi $pkgname-fpm $pkgname-pear::noarch
+	pdo_odbc pdo_pgsql pdo_sqlite pgsql phar posix pspell recode session shmop snmp soap
+	sockets sqlite3 sysvmsg sysvsem sysvshm tidy wddx xml xmlreader xmlrpc xmlwriter xsl zip zlib
 	"
-for _ext in $_exts; do
-	case "$_ext" in
-	*:*) subpackages="$subpackages $pkgname-$_ext";;
-	*) subpackages="$subpackages $pkgname-$_ext:_package_ext";;
-	esac
+for _extension in $_extensions; do
+	[ -z "${_extension##*:*}" ] && eval _index_${_extension%:*}=${_extension#*:}
+	subpackages="$subpackages $pkgname-${_extension%:*}:_extension"
 done
-subpackages="$subpackages $pkgname-common::noarch"
-
-_apiver="20151012"
+_prefix_opcache="zend_"
+_pkgdesc_opcache="Zend OPcache"
+subpackages="$pkgname-dev $pkgname-doc $pkgname-libs $pkgname-apache2 $pkgname-phpdbg
+	$pkgname-litespeed $pkgname-cgi $pkgname-fpm $pkgname-pear::noarch
+	$pkgname-phar-utils:_phar:noarch $subpackages $pkgname-config::noarch"
+builddir="$srcdir/$_pkgreal-$pkgver"
 
 prepare() {
 	cd "$builddir"
 
+	sed "$(($(grep -n 'Pool Definitions' sapi/fpm/php-fpm.conf.in | cut -d: -f1)+3)),\$d" \
+		-i sapi/fpm/php-fpm.conf.in
+	echo include=@php_fpm_sysconfdir@/php-fpm.d/*.conf >> sapi/fpm/php-fpm.conf.in
```

Comment:
This is quite cryptic and moreover totally unnecessary, because `include=@php_fpm_sysconfdir@/php-fpm.d/*.conf` is already there in 7.0.16 (the version specified in this exact revision) and newer.